### PR TITLE
Export OcTree

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
@psaripp Could please look into why this doesn't build. You are the original author of this code if I am correct.

**Comment by MatthijsBurgh** [comment]
@jihoonl Please be thorough on your review. I am not that sure about my work this time.

**Comment by jihoonl** [comment]
I agree with renaming to OccupancyMap to OcTree. It looked all good to me with a quick look. :wink:  
@MatthijsBurgh could you elaborate what your concern is?

@psaripp there are Octree module updates introduced in this PR. Would you double-check if it actually changes behavior of your code?

**Comment by MatthijsBurgh** [comment]
@jihoonl especially the transpiler and the exports. I am not sure I should export more or less.

I don't think it is needed to export base classes that are only used inside other classes.

**Comment by MatthijsBurgh** [comment]
> @jihoonl especially the transpiler and the exports. I am not sure I should export more or less.
> 
> I don't think it is needed to export base classes that are only used inside other classes.

@jihoonl could you please take a look?

**Comment by trusktr** [comment]
Looks like you fixed the bug in es6-transpiler that I encountered in #438. I deleted es6-transpiler in that PR, it's not worth maintaining that.

**Comment by MatthijsBurgh** [comment]
@trusktr I will merge this one. Can you update your PR(s) if needed?

File: src/navigation/OcTreeBase.js [hunk 1]
```diff
@@ -407,13 +434,13 @@ ROS3D.OcTreeBase.prototype._buildFaces = function () {
     _insertFace: function (face, pos, size, color) {
       const indexCount = this.vertices.length / 3;
 
-      for (let vertex of face.vertices) {
+      face.vertices.forEach(function(vertex) {
```
Comment (trusktr) [inline src/navigation/OcTreeBase.js:437]: Why change all the for-of loops to forEach?

Comment (MatthijsBurgh) [reply src/navigation/OcTreeBase.js:437]: I think it was producing linting warnings/errors.

Comment (trusktr) [reply src/navigation/OcTreeBase.js:437]: Fixed in the other PR

**Pull request merged**